base/system/chkdsk/chkdsk.c

```diff
@@ -406,7 +398,7 @@ wmain(int argc, WCHAR *argv[])
     {
         if (!GetCurrentDirectoryW(ARRAYSIZE(CurrentDirectory), CurrentDirectory))
         {
-            PrintWin32Error(L"Could not get current directory", GetLastError());
+            PrintWin32Error((LPWSTR)IDS_NO_CURRENT_DIR, GetLastError());
```

Comment:
Maybe something like this would suffice?
```C
static VOID PrintWin32Error(int Message, DWORD ErrorCode)
{
    ConResPrintf(StdErr, Message);
    ConMsgPuts(StdErr, FORMAT_MESSAGE_FROM_SYSTEM,
               NULL, ErrorCode, LANG_USER_DEFAULT);
    ConPuts(StdErr, L"\n");
}
```
`Message` will be an integer instead of a long wide pointer string since the resource strings (in this case) are always integers.

The reason for that cast was because of a compiler error which expects a LPWSTR variable but has int. I understand such solution is not intuitive -- I do not have other solution to come up in mind unfortunately.